pkg/apis/crds/karpenter.sh_nodepools.yaml

```diff
@@ -109,7 +109,7 @@ spec:
                       type: array
                       x-kubernetes-validations:
                         - message: '''schedule'' must be set with ''duration'''
-                          rule: '!self.all(x, (has(x.schedule) && !has(x.duration)) || (!has(x.schedule) && has(x.duration)))'
+                          rule: self.all(x, has(x.schedule) == has(x.duration))
```

Comment:
Shouldn't this be an `&&` and not an equal?

Reply:
`==` means they match. That's all that we require. Either they are both set or they are both not set.